tests/foreman/cli/test_activationkey.py

```diff
@@ -46,6 +44,8 @@
 from robottelo.constants import PRDS
 from robottelo.constants import REPOS
 from robottelo.constants import REPOSET
+from robottelo.host_helpers.cli_factory import CLIFactory
+from robottelo.host_helpers.cli_factory import CLIFactoryError
```

Comment:
Remove this helper from here and move under `robottelo/exceptions.py` module and import from there if not already moved.